crates/core/src/node/call_error_tracer.rs

```diff
@@ -24,37 +28,7 @@ impl<S, H: HistoryMode> DynTracer<S, SimpleMemory<H>> for CallErrorTracer {
         _memory: &SimpleMemory<H>,
     ) {
         if !data.error_flags_accumulated.is_empty() {
-            tracing::error!("!! Got error flags: ");
-            if data
-                .error_flags_accumulated
-                .contains(ErrorFlags::INVALID_OPCODE)
-            {
-                tracing::error!("INVALID OPCODE");
-            }
-            if data
-                .error_flags_accumulated
-                .contains(ErrorFlags::NOT_ENOUGH_ERGS)
-            {
-                tracing::error!("NOT ENOUGH ERGS");
-            }
-            if data
-                .error_flags_accumulated
-                .contains(ErrorFlags::PRIVILAGED_ACCESS_NOT_FROM_KERNEL)
-            {
-                tracing::error!("PRIVILEGED ACCESS");
-            }
-            if data
-                .error_flags_accumulated
-                .contains(ErrorFlags::WRITE_IN_STATIC_CONTEXT)
-            {
-                tracing::error!("WRITE IN STATIC");
-            }
-            if data
-                .error_flags_accumulated
-                .contains(ErrorFlags::CALLSTACK_IS_FULL)
-            {
-                tracing::error!("CALLSTACK IS FULL");
-            }
+            let _ = self.result.set(data.error_flags_accumulated);
```

Comment:
Do I understand correctly that we rely on top frame to be called last here? Thus overwriting any other error flags that might have been observed before

Reply:
Yes that is correct which I think is the behaviour we want? Would there be a case where multiple unique flags could be observed at different points? I didnt think that would be the case. 

If so, I can update to accumulate all unique flags observed.

Reply:
Yes, I believe this is the behaviour we want. I asked because it is not immediately obvious from the code. IMO it would be good to add a comment just so no one accidentally changes the behaviour in the future.